entries/QUnit.module.xml

```diff
@@ -45,11 +60,33 @@ QUnit.module( "group b" );
 QUnit.test( "a basic test example 3", function( assert ) {
 	assert.ok( true, "this test is fine" );
 });
-QUnit.test( "a basic test example 4", function() {
+QUnit.test( "a basic test example 4", function( assert ) {
 	assert.ok( true, "this test is fine" );
 });
 ]]></code>
 	</example>
+		<example>
+		<desc>Use the <code>QUnit.module()</code> function to group tests together:</desc>
+<code><![CDATA[
+QUnit.module( "module a", function( test ) {
+	test( "a basic test example", function( assert ) {
+		assert.ok( true, "this test is fine" );
+	});
+});
+
+QUnit.module( "module b", function( test ) {
+	test( "a basic test example 2", function( assert ) {
+		assert.ok( true, "this test is fine" );
+	});
+
+	QUnit.module( "nested module b.1", function( test ) {
+		test( "a basic test example 3", function( assert ) {
```

Comment:
Let's add comments that describe the resulting "test name", to explain how nested modules and test names are combined.